replica-common/src/main/java/com/graphhopper/CustomGraphHopperGtfs.java

```diff
@@ -173,6 +179,15 @@ public void writeOsmNodeIds(Map<Integer, Long> ghToOsmNodeIds) {
         }
     }
 
+    public void writeGhEdgeIdToSegmentIndexes(Map<Integer, Integer> ghEdgeIdToSegmentIndex) {
+        for (int ghEdgeId : ghEdgeIdToSegmentIndex.keySet()) {
+            int segmentIndex = ghEdgeIdToSegmentIndex.get(ghEdgeId);
+            long pointer = 4L * ghEdgeId;
```

Comment:
Note: we know these are short ints so we only use 4 bits to store them instead of 8 like the other custom info we store

Reply:
Did I say bits? Oops...I meant bytes 🙃 glad you asked though, I sanity checked via IDE and took a very deep breath when I realized I was wrong lol
<img width="471" alt="Screen Shot 2023-02-23 at 3 05 53 PM" src="https://user-images.githubusercontent.com/13446427/221051320-a2024b13-cfdc-45d5-abb2-8f2e7d6c25ab.png">

Reply:
so I think we're good, unless a way has 2^31 segments